### ISSUE TITLE
Module `eve` not found

### ISSUE THREAD

**Issue by BlockChange** [geelen/Snap.svg-cjs#3]
When trying:

`import Snap from 'snapsvg-cjs';`

Then:

```
 var s = Snap("#elementID");
 var bigCircle = s.circle(150, 150, 100);
```

I get:
Cannot find module "snapsvg/node_modules/eve"

**Comment by codeimpossible** [comment]
@BlockChange check out https://github.com/geelen/Snap.svg-cjs/pull/2 looks like this might be an issue with using snapsvg-cjs with npm3?

**Comment by geelen** [comment]
Closed in `v0.0.4` thanks!
